operations/compare-helm-with-jsonnet/plugins/resolve-config/main.go

```diff
@@ -0,0 +1,349 @@
+package main
```

Comment:
would be nice to document what this program does. A short readme should suffice

Reply:
Agreed, I'll take care of it